### ISSUE TITLE
设置 resume_checkpoint 时出错 2022-12-08 13:10:43 [ERROR]     pretrain_weights and resume_checkpoint cannot be set simultaneously.

### ISSUE THREAD

**Issue by mzb157** [PaddlePaddle/PaddleX#1638]
## 描述问题
这是我的训练代码，把  pretrain_weights 写成None了依然报错pretrain_weights and resume_checkpoint cannot be set simultaneously.

#代码
import os

os.environ['CUDA_VISIBLE_DEVICES'] = '0'

import paddlex as pdx
from paddlex import transforms as T

train_transforms = T.Compose([
   T.MixupImage(mixup_epoch=-1), T.RandomDistort(),
   T.RandomExpand(im_padding_value=[123.675, 116.28, 103.53]), T.RandomCrop(),
   T.RandomHorizontalFlip(), T.BatchRandomResize(
       target_sizes=[
           320, 352, 384, 416, 448, 480, 512, 544, 576, 608, 640, 672, 704,
           736, 768
       ],
       interp='RANDOM'), T.Normalize(
           mean=[0.485, 0.456, 0.406], std=[0.229, 0.224, 0.225])
])

eval_transforms = T.Compose([
    T.Resize(
        target_size=640, interp='CUBIC'), T.Normalize(
            mean=[0.485, 0.456, 0.406], std=[0.229, 0.224, 0.225])
])

train_dataset = pdx.datasets.VOCDetection(
    data_dir='../fire_smoke/',
    file_list='../fire_smoke/train.txt',
    label_list='../fire_smoke/label_list.txt',
    transforms=train_transforms,
    shuffle=True)
eval_dataset = pdx.datasets.VOCDetection(
    data_dir='../fire_smoke/',
    file_list='../fire_smoke/valid.txt',
    label_list='../fire_smoke/label_list.txt',
    transforms=eval_transforms,
    shuffle=False)

num_classes = len(train_dataset.labels)
model = pdx.det.PPYOLOv2(num_classes=num_classes,)#, backbone='ResNet50_vd_dcn')
model.train(
    num_epochs=270,
    train_dataset=train_dataset,
    train_batch_size=4,
    eval_dataset=eval_dataset,
    learning_rate=0.005 / 12,
    warmup_steps=1000,
    warmup_start_lr=0.0,
    lr_decay_epochs=[105, 135, 150],
    save_interval_epochs=5,
    save_dir='output/ppyolov2_r50vd_dcn_aug',
    resume_checkpoint='output/ppyolov2_r50vd_dcn_aug/epoch_30/',
    pretrain_weights = 'None'
    )

#报错：
2022-12-08 13:10:43 [ERROR]     pretrain_weights and resume_checkpoint cannot be set simultaneously.

**Comment by lailuboy** [comment]
多了引号，pretrain_weights = None 或者 这个参数去掉。
